src/object/coin.cpp

```diff
@@ -234,6 +235,12 @@ HeavyCoin::update(float dt_sec)
   m_col.set_movement(m_physic.get_movement(dt_sec));
 }
 
+void
+HeavyCoin::draw(DrawingContext& context)
+{
+  m_sprite->draw(context.color(), get_pos(), m_layer);
+}
+
```

Comment:
What is the redefinition for? Is there a difference with `MovingSprite::draw`?

Reply:
I'll have a look when I have time. `HeavyCoin`, which is gravity-affected and doesn't flip, inherits from `Coin`, which does. I know I changed the way that is handled at least twice when I was coding this, so I will double-check again.

Reply:
Turns out I did that to not pass the `m_flip` variable to the draw function (to make it default to NO_FLIP), but since I then changed it so `m_flip` never gets transformed for `HeavyCoin`s, it makes this override unneeded. I'll remove it.